pkg/trisa/gds/api/v1beta1/api_test.go

```diff
@@ -0,0 +1,26 @@
+package api_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	api "github.com/trisacrypto/trisa/pkg/trisa/gds/api/v1beta1"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+func TestRegistrationForm(t *testing.T) {
```

Comment:
This test loads the JSON saved from the GDS UI form to ensure the new protocol buffers match the expected structure.